types/react/index.d.ts

```diff
@@ -2789,7 +2789,7 @@ declare namespace React {
         name?: string;
         required?: boolean;
         size?: number;
-
+        value?: string | string[] | number;
```

Comment:
In the case of a Select, it might be in  `<select multiple>`, I simply kept the definition that existed before and distributed it to form fields